src/WebUI/Pods/PodsDetails.tsx

```diff
@@ -71,5 +83,20 @@ export class PodsDetails extends BaseComponent<IPodsDetailsProperties, IPodsDeta
         });
     }
 
+    private _showImageDetails = (imageId: string) => {
+        const imageService = KubeSummary.getImageService();
```

Comment:
a thought, should we have a store and then use that to get the data?

Reply:
Added a ToDO for the same